scripts/superdesk/datetime/datetime.js

```diff
@@ -73,6 +73,42 @@
         }
     }
 
+    DateTimeHelperService.$inject = ['moment', 'config'];
+    function DateTimeHelperService(moment, config) {
+
+        /*
+        * @param timestring 2016-03-01T04:45:00+0000
+        * @param timezone Europe/London
+        */
+        this.splitDateTime = function(timestring, timezone) {
+            var momentTS = moment.tz(timestring, timezone);
+
+            return {
+                'date': momentTS.format(config.model.dateformat),
```

Comment:
that's what is changed in PR, the time won't be utc anymore, but it will be local according to timezone